lib/tapioca/commands/annotations.rb

```diff
@@ -125,7 +125,7 @@ def fetch_annotations(project_gems)
         fetched_gems.keys.map(&:name).sort
       end
 
-      sig { params(repo_uris: T::Array[String], gem_info: GemInfo).returns(T::Boolean) }
+      #: (Array[String] repo_uris, GemInfo gem_info) -> bool
```

Comment:
Out of curiosity: Why was this not translated before? It doesn't seem to be inside a block.